imageio/core/format.py

```diff
@@ -350,6 +350,17 @@ def get_next_data(self, **kwargs):
             listed in the documentation of those formats.
             """
             return self.get_data(self._BaseReaderWriter_last_index+1, **kwargs)
+
+        def set_image_index(self, index, **kwargs):
+            """ set_image_index(index)
+
+            Set the pointer to the last read index such that the next call to
+            get_next_data() returns the image specified by the index
```

Comment:
Can you please rephrase this a bit? It contains "internal detail" which will only confuse users. Something like "Set the internal pointer to the given index, such that the next ..." should be enough.